### ISSUE TITLE
using the original labels, instead of inverted transformed labels for evaluation

### ISSUE THREAD

**Issue by wyli** [Project-MONAI/model-zoo#181]
Currently quite a few bundles use 
the inverted version of preprocessed original labels as the reference during evaluation:

https://github.com/Project-MONAI/model-zoo/blob/987f49fe9e0f20f047a2252c1ace38bd760cfb33/models/spleen_ct_segmentation/configs/evaluate.json#L12-L21

which is different from the approach taken in the tutorial using the original label directly during evaluation (https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb)

<img src=https://user-images.githubusercontent.com/831580/200014448-54076dc9-e510-40fa-adca-ae5f3efa8bcb.png width=800></img>

would be great to consider the standard method used in the tutorial for bundle evaluate.json

cc @holgerroth @dongyang0122 

(this needs an update as well) https://github.com/Project-MONAI/tutorials/tree/main/bundle/spleen_segmentation)

**Comment by dongyang0122** [comment]
It is necessary to validate without modifying the label data for (almost) all models in the model zoo.

**Comment by yiheng-wang-nv** [comment]
Hi @Nic-Ma @wyli , to avoid maintaining duplicate spleen bundles in tutorial and model zoo, should we remove the one in the tutorial, and add a step in the readme file of tutorial to do "download from model zoo"?

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv ,

I think that's OK, and just put a link to the source code of spleen bundle in the model-zoo.
@wyli What do you think?

Thanks.

**Comment by wyli** [comment]
sure, i think it'll be a separate new feature request in the tutorial repo, or is it related to this issue?

**Comment by wyli** [comment]
(this `device='cpu'` fix will be needed for `Invertd` https://github.com/Project-MONAI/tutorials/pull/1042 for the monai v1.1 bundles)

**Comment by yiheng-wang-nv** [comment]
> sure, i think it'll be a separate new feature request in the tutorial repo, or is it related to this issue?

Yes, it should be a separate new feature request in the tutorial repo, I just created one: https://github.com/Project-MONAI/tutorials/issues/1043

As for the issue of using inverted labels, may need further discussions with @Nic-Ma @daguangxu @dongyang0122 @wyli

**Comment by yiheng-wang-nv** [comment]
For the issue, I checked and the following bundles need to be updated:
1. `spleen_ct_segmentation` (invert transformed labels for evaluation)
2. `endoscopic_tool_segmentation` (no invert in postprocessing)
3. `pancreas_ct_dints_segmentation` (invert transformed labels for evaluation)
4. `spleen_deepedit_annotation` (no invert in postprocessing)
5. `swin_unetr_btcv_segmentation` (invert transformed labels for evaluation)

**Comment by mingxin-zheng** [comment]
> (this `device='cpu'` fix will be needed for `Invertd` [Project-MONAI/tutorials#1042](https://github.com/Project-MONAI/tutorials/pull/1042) for the monai v1.1 bundles)

Hi @wyli , I don't fully understand this requirement of `device='cpu'`. Would you mind explaining a bit more, for example, if we don't set it to `cpu`, will it break something in the spleen segmentation tutorial?

**Comment by wyli** [comment]
yes, it will raise this error when computing metrics: https://github.com/Project-MONAI/tutorials/issues/1041

**Comment by mingxin-zheng** [comment]
@wyli Thanks!

**Comment by yiheng-wang-nv** [comment]
> For the issue, I checked and the following bundles need to be updated:
> 
> 1. `spleen_ct_segmentation` (invert transformed labels for evaluation)
> 2. `endoscopic_tool_segmentation` (no invert in postprocessing)
> 3. `pancreas_ct_dints_segmentation` (invert transformed labels for evaluation)
> 4. `spleen_deepedit_annotation` (no invert in postprocessing)
> 5. `swin_unetr_btcv_segmentation` (invert transformed labels for evaluation)

Update: to prevent unexpected errors, this issue will be addressed after our next release.

**Comment by yiheng-wang-nv** [comment]
Update: for 5 bundles mentioned above, only `spleen_deepedit_annotation` is still missing the invert step in evaluation config now and I will update it when addressing #362
